app/src/actions/map.js

```diff
@@ -1,13 +1,20 @@
-import PelagosClient from '../lib/pelagosClient';
 import _ from 'lodash';
+import PelagosClient from '../lib/pelagosClient';
+import { push } from 'react-router-redux';
+
 import {
   SET_LAYERS,
   SET_ZOOM,
   SET_CENTER,
   TOGGLE_LAYER_VISIBILITY,
   SET_TIMELINE_DATES,
-  GET_SERIESGROUP
+  GET_SERIESGROUP,
+  SHARE_MODAL_OPEN,
+  SET_WORKSPACE_ID,
+  DELETE_WORKSPACE_ID,
+  SET_SHARE_MODAL_ERROR
 } from '../constants';
+
 const urlVessel = 'https://skytruth-pleuston.appspot.com/v1/tilesets/tms-format-2015-2016-v1/sub/';
```

Comment:
The PR wasn't aimed to refactor the rest of the file. This bit is for another action.
Definitely agree it should be moved.